docs/content/library/AsyncArrow.fsx

```diff
@@ -0,0 +1,339 @@
+(**
+
+# F# Async: AsyncArrow
+
+An `AsyncArrow<'a, 'b>` is a function which takes a value of type `'a` as input and returns
+and async computation which produces a value of type `'b`. An async arrow is represented
+as a type alias `type AsyncArrow<'a, 'b> = 'a -> Async<'b>`. An async arrow can represent an 
+async request-reply protocol and therefore operations on arrows are operations on said protocols.
+Many of the operations defined on ordinary functions, such as composition `>>` can also be 
+defined for async arrows by simply lifting them into the `Async` type with operators defined therein.
+
+Async arrows are useful beyond the `Async` type alone when there is a need to consider the input
+which creates the async computation. Arrows also capture common ways of *gluing* together async request-reply
+protocols. Furthermore, mappings between async arrows, called async filters, can also be useful to consider as a 
+reified type for implementing service pipelines.
+
+The `AsyncArrow` type is located in the `FSharpx.Async.dll` assembly which can be loaded in F# Interactive as follows:
+*)
+
+#r "../../../bin/FSharpx.Async.dll"
+#r "System.Net.Http.dll"
+open FSharpx.Control
+
+
+(**
+## Motivation
```

Comment:
A section could be added here on "If you know F# Async, then you've already met some asynchronous functions" and then give examples.